### PR TITLE
Fix import in BooleanFilter.js

### PR DESCRIPTION
**Pull request by vmihalachi**
It was my mistake in the last PR, sorry about that. It is missing an import in this file. 

Can you help me understanding how to test the changes to this project? Perhaps we can run the tests during the build :)

### CODE REVIEW

File: src/components/MTableFilterRow/BooleanFilter.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Checkbox } from '@material-ui/core';
+import { FormControlLabel, Checkbox } from '@material-ui/core';
```
Comment (matthewoestreich) [inline src/components/MTableFilterRow/BooleanFilter.js:2]: Isn't `FormControlLabel` being used, though?

https://github.com/material-table-core/core/blob/e2992cac88778052a8896a8267f5d80f8d82aa6e/src/components/MTableFilterRow/BooleanFilter.js#L1-L6

**Comment by thefat32** [comment]
Looks fine

**Review by thefat32 — APPROVED** [review]

**Review by thefat32 — APPROVED** [review]

**Review by thefat32 — APPROVED** [review]

**Comment by thefat32** [comment]
It seems I don't have write permission, please @oze4 merge this

**Comment by thefat32** [comment]
@vmihalachi well, we need to write more jest tests to prevent this from happening as `no-undef` is a warning in eslint

**Comment by matthewoestreich** [comment]
But why is that import being removed when we are using it? I don't follow.

**Comment by matthewoestreich** [comment]
Absolutely more tests are needed. We can use all the help we can get. 

I am focusing on refactoring components to functional components (not on ts)  before starting tests.

**Comment by thefat32** [comment]
@oze4 this PR is adding missing FormControlLabel import from old @vmihalachi PR

**Comment by matthewoestreich** [comment]
Oh geez lol I read that wrong.

**Review by matthewoestreich — APPROVED** [review]

**Pull request merged**